Testing/Heroes/VanishTests.cs

```diff
@@ -548,15 +548,15 @@ public void FlashRecon()
         [Test]
         public void FlashReconOblivAeon()
         {
-            SetupGameController(new string[] { "OblivAeon", "Cauldron.Vanish", "Legacy", "Haka", "Cauldron.WindmillCity", "MobileDefensePlatform", "InsulaPrimalis", "Cauldron.VaultFive", "Cauldron.Northspar" }, shieldIdentifier: "PrimaryObjective");
+            SetupGameController(new string[] { "OblivAeon", "Cauldron.Vanish", "Legacy", "Haka", "Cauldron.WindmillCity", "MobileDefensePlatform", "InsulaPrimalis", "Cauldron.VaultFive", "Cauldron.Northspar" }, shieldIdentifier: "ThePrimaryObjective");
             StartGame();
             GoToStartOfTurn(vanish);
 
             Card aeonWarrior = GetCard("AeonWarrior");
             PlayCard(oblivaeon, aeonWarrior, overridePlayLocation: vanish.BattleZone.FindScion().PlayArea);
-            PlayCard(oblivaeon, borrScion, overridePlayLocation: vanish.BattleZone.FindScion().PlayArea);
-
+            //PlayCard(oblivaeon, borrScion, overridePlayLocation: vanish.BattleZone.FindScion().PlayArea);
```

Comment:
What's up with commenting this out?

Reply:
The purpose of that line in the original test was to put a Scion in BZ1 so that the Scion deck is visible to Flash Recon. By fixing the SetupGameController string so that the shield gets put in play, it ensures that Voidsoul will be in BZ1, and playing another scion is unnecessary.